lua/telescope/config.lua

```diff
@@ -151,6 +151,21 @@ append(
   - "ascending"]]
 )
 
+append(
+  "tiebreak",
+  function(current_entry, existing_entry, _)
+    return #current_entry.ordinal < #existing_entry.ordinal
+  end,
+  [[
+  tiebreak can be set to a function which decides how to break
```

Comment:
\`tiebreak\`
I think we typically have full sentences ending with a full stop in comparable docs here.

when two entries (sorry, OCD-level nit).

Let's clarify that returning `true` means the current entry is placed before the existing entry (IIRC).